CodeForBaltimore/JuryInstructions
# adding pdf and docx generation; cleanup of the first page

# Description

This will add PDF and docx file generation to the final page. Additionally, it adds a basic text editor to the last page.

## Related Issues

List related Issues:

| link |
| ---- |
|  #13   |

## Todos

- [x] docx file creation
- [x] pdf generation
- [x] text editor

## Steps to Test or Reproduce

Start the app and then fill out some data on the first page (make sure Homicide, at least, is selected). Click submit on the second page with no changes. Make any desired change to the text on the last page and then hit either export button.

## Impacted Areas in Application

List general components of the application that this PR will affect:

- the last page.
